services/api_gateway/src/main/java/com/futurewei/alcor/apigateway/debug/DebugWebConfiguration.java

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+
+package com.futurewei.alcor.apigateway.debug;
```

Comment:
does Java problem have "debug" mode as a compile option, maybe we can put the debug stuff under the "debug" mode?

Reply:
This 'debug' controller could be potentially used by the admin to get some extra information. I think we could use a better name like 'admin' instead of 'debug'.